packages/next/src/build/webpack/plugins/build-manifest-plugin.ts

```diff
@@ -168,6 +168,74 @@ export const processRoute = (r: Rewrite) => {
   return rewrite
 }
 
+/* Inspired by `next/src/build/webpack/react-loadable-plugin.ts` */
+const addDynamicCSSToBuildManifest = ({
+  compilation,
+  block,
+  assetMap,
+}: {
+  compilation: webpack.Compilation
+  block: webpack.AsyncDependenciesBlock
+  assetMap: DeepMutable<BuildManifest>
+}) => {
+  block.blocks.forEach((recursiveBlock) =>
+    addDynamicCSSToBuildManifest({
+      compilation,
+      block: recursiveBlock,
+      assetMap,
+    })
+  )
+
+  const dynamicCSSFiles = new Set<string>()
```

Comment:
Same question like last round, should this go under the async dependency checking?

We need a test case to cover thise: if we import a css in a sync dependency, it won't be collected as dynamic css